share/config_settings.toml

```diff
@@ -0,0 +1,145 @@
+        default = 22
+
+        [security.ssh.ssh_password_authentication]
+        type = "boolean"
+        yes = "True"
+        no = "False"
+        default = "False"
+
+        [security.ssh.ssh_allow_deprecated_dsa_hostkey]
+        type = "boolean"
+        yes = "True"
+        no = "False"
+        default = "False"
+
+    [security.nginx]
+    name = "NGINX"
+        [security.nginx.nginx_redirect_to_https]
+        type = "boolean"
+        yes = "True"
+        no = "False"
+        default = "True"
+
+        [security.nginx.nginx_compatibility]
+        type = "select"
+        default = "intermediate"
+        choices = ["intermediate", "modern"]
+
+    [security.postfix]
+    name = "Postfix"
+        [security.postfix.postfix_compatibility]
+        type = "select"
+        default = "intermediate"
+        choices = ["intermediate", "modern"]
+
+    [security.webadmin]
+    name = "Webadmin"
+        [security.webadmin.webadmin_allowlist_enabled]
+        type = "boolean"
+        yes = "True"
+        no = "False"
+        default = "False"
+
+        [security.webadmin.webadmin_allowlist]
+        type = "tags"
+        visible = "webadmin_allowlist_enabled"
+        optional = true
+        default = ""
+
+    [security.experimental]
+    name = "Experimental"
+        [security.experimental.security_experimental_enabled]
+        type = "boolean"
+        yes = "True"
+        no = "False"
+        default = "False"
+
+
+[email]
+name = "Email"
+    [email.pop3]
+    name = "POP3"
+        [email.pop3.pop3_enabled]
+        type = "boolean"
+        yes = "True"
+        no = "False"
+        default = "False"
+    
+    [email.smtp]
+    name = "SMTP"
+        [email.smtp.smtp_allow_ipv6]
+        type = "boolean"
+        yes = "True"
+        no = "False"
+        default = "True"
+
+        [email.smtp.smtp_relay_enabled]
+        type = "boolean"
+        yes = "True"
+        no = "False"
+        default = "False"
+        
+        [email.smtp.smtp_relay_host]
+        type = "string"
+        default = ""
+        optional = true
+        visible="smtp_relay_enabled"
+
+        [email.smtp.smtp_relay_port]
+        type = "number"
+        default = 587
+        visible="smtp_relay_enabled"
+
+        [email.smtp.smtp_relay_user]
+        type = "string"
+        default = ""
+        optional = true
+        visible="smtp_relay_enabled"
+        
+        [email.smtp.smtp_relay_password]
+        type = "password"
+        default = ""
+        optional = true
+        visible="smtp_relay_enabled"
+
+[misc]
+name = "Other"
+    [misc.ssowat]
+    name = "SSOwat"
+        [misc.ssowat.ssowat_panel_overlay_enabled]
+        type = "boolean"
+        yes = "True"
+        no = "False"
+        default = "True"
+    
+    [misc.backup]
+    name = "Backup"
+        [misc.backup.backup_compress_tar_archives]
+        type = "boolean"
+        yes = "True"
+        no = "False"
+        default = "False"
```

Comment:
So I guess in the mid/long-term we'll probably want a `portal` and `backup` entire sections (or is that a panel ?). But maybe migrating the name later is not really a big deal (thinking out loud again)